cyclonedx/exception/output.py

```diff
@@ -22,9 +22,10 @@
 from . import CycloneDxException
 
 
-class ComponentVersionRequiredException(CycloneDxException):
```

Comment:
was this a public class? if so, the removal will cause a breaking change for some people downstream

Reply:
It was a public class @jkowalleck. Agree this is a breaking change.